index.json

```diff
@@ -105,6 +105,9 @@
   "beatoff": 2,
   "beatyourmeat": 2,
   "beaver": 0,
+  "bellend": 2,
+  "bellends": 2,
+  "bellendz": 1,
```

Comment:
Why are singular and plural rated as `2`, whereas the “leetspeek”-like version is `1`?